projects/netgrif-components/src/lib/header/header.component.html

```diff
@@ -3,41 +3,46 @@
         <div fxLayout="row" fxFlex="95" class="header-content-container flex-95" fxLayoutAlign="start center"
              [ngSwitch]="headerService.headerState.mode" *ngIf="!headerService.loading.isActive">
             <div *ngSwitchCase="headerModeEnum.SORT" fxFlex="100" fxLayout="row">
-                <nc-sort-mode fxFlex="100" [headerService]="headerService" [overflowWidth]="getMinWidth()"></nc-sort-mode>
+                <nc-sort-mode fxFlex="100" [headerService]="headerService"
+                              [overflowWidth]="getMinWidth()"></nc-sort-mode>
```

Comment:
can these widths be set with a pipe? Should have better performance

Reply:
create new task